tasks/gotest.py

```diff
@@ -833,7 +834,8 @@ def should_run_all_tests(files, trigger_files):
 
 
 def get_go_modified_files(ctx):
-    files = get_modified_files(ctx)
+    base_branch = _get_release_json_value("base_branch")
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
I see that `_get_release_json_value` can raise an error, currently exiting the process. Wonder if we should raise an error and deal with it here, or if it is safe to consider we always have a `base_branch`

Reply:
I think this is rather safe to consider it will not fail because `base_branch` must always exist [here](https://github.com/DataDog/datadog-agent/blob/main/release.json#L2)
If it does not this is likely to break other part that use this value